### ISSUE TITLE
Add lndir to standard containers (el7, el8, el9)

### ISSUE THREAD

**Issue by kpedro88** [cms-sw/cms-docker#350]
The `lndir` utility is often useful for various purposes and has typically been included in host OS distributions. Can it be added to the standard CMS containers?

**Comment by smuzaffar** [comment]
@kpedro88 , `lndir` unitlity if provided by `imake` package [a]. Looks like is it deprecated and only available for almalinux9. 

[a]
```
Name         : imake
Version      : 1.0.8
Release      : 6.el9
Architecture : x86_64
Size         : 1.2 M
Source       : imake-1.0.8-6.el9.src.rpm
Repository   : @System
From repo    : epel
Summary      : imake source code configuration and build system
URL          : http://www.x.org
License      : MIT
Description  : Imake is a deprecated source code configuration and build system which
             : has traditionally been supplied by and used to build the X Window System
             : in X11R6 and previous releases.  As of the X Window System X11R7 release,
             : the X Window system has switched to using GNU autotools as the primary
             : build system, and the Imake system is now deprecated, and should not be
             : used by new software projects.  Software developers are encouraged to
             : migrate software to the GNU autotools system.
```

**Comment by kpedro88** [comment]
Fermilab and lxplus have `imake-1.0.7-11.el8.x86_64` on the EL8 nodes and `imake-1.0.8-6.el9.x86_64` on the EL9 nodes.

I tried running `yum install imake` in a Docker build from base `sl:7`. It finds `1.0.5-10.el7` and `lndir` gets installed. So I think we can just add this to the dockerfile. (In some other OSes, `lndir` is distributed as part of `xorg`, which hosts it at https://gitlab.freedesktop.org/xorg/util/lndir. As usual, RHEL has to be special...)

**Comment by smuzaffar** [comment]
@kpedro88 , I was not able to install it from epel or other yum repos. If you know how to do it then please open a PR.

**Comment by kpedro88** [comment]
@smuzaffar it appears to come from the sl repo at `http://ftp.scientificlinux.org/linux/scientific/obsolete`. Is it okay to add this repo to the list in our el7 container? If so, I will go ahead and prepare a PR after some more testing.

**Comment by smuzaffar** [comment]
@kpedro88 , yes it is ok to add this repo but please do it at the end so that all other packages still gets installed from epel/cern repos.

**Comment by kpedro88** [comment]
Finally addressed in #374
